api/MediaKeySystemConfiguration.json

```
@@ -63,10 +63,10 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "44"
```

```suggestion
              "version_added": "43"
```

I was distrusting of https://bugzilla.mozilla.org/show_bug.cgi?id=1189196 since there are parts fixed in 43 and 44. I don't know if there's a tool like https://omahaproxy.appspot.com/ for Firefox, but this confirms this was in 43:
https://hg.mozilla.org/releases/mozilla-release/file/FIREFOX_43_0_RELEASE/dom/webidl/MediaKeySystemAccess.webidl#l21

https://hg.mozilla.org/releases/mozilla-release/file/FIREFOX_43_0_RELEASE/dom/webidl/MediaKeySystemAccess.webidl